### PR TITLE
Awesome books One-Page navigation

### PR DESCRIPTION
**Pull request by okoyecharles**
## Awesome books: a complete website with navigation
In this project, we took the Awesome books application and converted it into a fully functional website. We refactored the project to add three new sections on top of the two sections we had already completed: 
- Books list
- Add a book form
- A New Contact info section, a Navigation bar, and a Footer.

## Changes Made
- [x] Adding a working Navigation Menu.
- [x] Creating three sections for each
- [x] Using One-Page Navigation
- [x] Fixing all Linter Errors

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -151,3 +151,14 @@ const removeBook = () => {
 document.addEventListener('click', () => {
   removeBook();
 });
```
Comment (medaminedev66) [inline js/script.js:153]: - [ ] The remove button does not function as intended. I tried to remove a book by clicking on the remove icon, but the book was not removed from the UI and the local storage. Please work on that.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @OkoyeCharles and @Owusu-Desmond  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- There are no linter errors ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/script.js (hunk 1 above)
Comment (Owusu-Desmond) [reply js/script.js:153]: Hi @medaminedev66 

Please before you start our application make sure your `localStorage` for the page do not contains a key of `books`
Please if you are still not okay with that drop your contact details and make we show you that everything is working our side.

**Review by Mre55 — APPROVED** [review]
**Hi @OkoyeCharles,**

# Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 



**[OPTIONAL]** Kindly, try to make it efficient your `remove` is working but as not as intended.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**